server/middleware/authMiddleware.js

```diff
@@ -6,17 +6,16 @@ const protect = asyncHandler(async (req, res, next) => {
     let token;
 
     if (
-        req.headers.authorization &&
-        req.headers.authorization.startsWith("Bearer")
```

Comment:
Why did you remove this? Instead maybe change the authHeader to include the word Bearer (it gives other developers an idea of the type of token being sent)